Makefile

```diff
@@ -7,7 +7,7 @@ SRCS = %.go
 all: $(BINARY)
 
 $(BINARY): deps **/**/*.go **/*.go *.go
-	go build -ldflags "-X main.buildCommit `git describe --long --tags --dirty --always`" ./cmd/ssh-chat
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags "-X main.buildCommit `git describe --long --tags --dirty --always`" ./cmd/ssh-chat
```

Comment:
How about just a standalone script for building the docker image that we can run from travis or locally?